### PR TITLE
fix: test scripts, README; add: env variables, config

### PR DESCRIPTION
**Pull request by Ankit098**

### CODE REVIEW

File: scripts/local.php [hunk 1]
```diff
@@ -8,30 +9,38 @@
...
+
+    # read credentials from environment variables
+    $USERNAME = getenv('BROWSERSTACK_USERNAME');
+    $ACCESS_KEY = getenv('BROWSERSTACK_ACCESS_KEY');
+
+    # if not provided in env vars, read credentials from config file
+    if (!isset($USERNAME)) {
+        $USERNAME = constant('BROWSERSTACK_USERNAME');
+    }
+    if (!isset($USERNAME)) {
```
Comment (kamal-kaur04) [inline scripts/local.php:30]:
```suggestion
    if (!isset($ACCESS_KEY)) {
```

File: scripts/parallel.php [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
+
+    # read credentials from environment variables
+    $USERNAME = getenv('BROWSERSTACK_USERNAME');
+    $ACCESS_KEY = getenv('BROWSERSTACK_ACCESS_KEY');
+
+    # if not provided in env vars, read credentials from config file
+    if (!isset($USERNAME)) {
+        $USERNAME = constant('BROWSERSTACK_USERNAME');
+    }
+    if (!isset($USERNAME)) {
```
Comment (kamal-kaur04) [inline scripts/parallel.php:16]:
```suggestion
    if (!isset($ACCESS_KEY)) {
```

File: scripts/single.php [hunk 3]
```diff
@@ -1,29 +1,38 @@
...
+    # read credentials from environment variables
+    $USERNAME = getenv('BROWSERSTACK_USERNAME');
+    $ACCESS_KEY = getenv('BROWSERSTACK_ACCESS_KEY');
 
-    $web_driver = RemoteWebDriver::create("https://$BROWSERSTACK_USERNAME:$BROWSERSTACK_ACCESS_KEY@hub-cloud.browserstack.com/wd/hub",$caps);
+    # if not provided in env vars, read credentials from config file
+    if (!isset($USERNAME)) {
+        $USERNAME = constant('BROWSERSTACK_USERNAME');
+    }
+    if (!isset($USERNAME)) {
```
Comment (kamal-kaur04) [inline scripts/single.php:28]:
```suggestion
    if (!isset($ACCESS_KEY)) {
```

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

**Review by kamal-kaur04 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,65 +1,49 @@
 # php-selenium-browserstack
+Code samples for php selenium tests on browserstack.
 
 ## Prerequisites 
-1. `php` and `composer` should be installed in your system
-2. Go to the directory where you have your test scripts and run the command below:
+1. `php` and `composer` should be installed in your system.
+* If you don't already use Composer, you can download the composer.phar binary:
 ```
-  composer require php-webdriver/webdriver
```
Comment (karanshah-browserstack) [inline README.md:7]: Where are we doing this step now?

File: scripts/single.php [hunk 5]
```diff
@@ -43,4 +52,3 @@
         echo 'Message: ' .$e->getMessage();
     }
     $web_driver->quit();
-?>
```
Comment (karanshah-browserstack) [inline scripts/single.php:46]: Need to have php closing braces

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Ankit098) [reply README.md:7]: `php composer.phar install` takes care of it

File: scripts/single.php (hunk 5 above)
Comment (Ankit098) [reply scripts/single.php:46]: ack, added now.. somehow tests ran even with this missing.

**Review by karanshah-browserstack — APPROVED** [review]

**Pull request merged**